PED/SetEnableHandcuffs.md

```diff
@@ -9,7 +9,9 @@ void SET_ENABLE_HANDCUFFS(Ped ped, BOOL toggle);
 ```
 
 ```
-ped can not pull out a weapon when true  
+Ped cannot pull out a weapon
```

Comment:
Markdown [code blocks](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks) are reserved for nativedb imports. If you're adding documentation to a native, get rid of the code blocks altogether, here's a [link](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) to the contribution guide :-)

The reason for this is that it will mark the [native documentation](https://docs.fivem.net/natives/) as 'Has "imported" docs', when you actually contributed to it, when it should be 'Has "modern" docs' (when the description isn't left blank).

Thanks!